sites/public/src/components/account/ApplicationsView.tsx

```diff
@@ -0,0 +1,229 @@
+import Layout from "../../layouts/application"
+import { StatusItemWrapper, AppWithListing } from "./StatusItemWrapper"
+import { MetaTags } from "../shared/MetaTags"
+import { UserStatus } from "../../lib/constants"
+
+import styles from "./ApplicationsView.module.scss"
+import {
+  ListingsStatusEnum,
+  LotteryStatusEnum,
+} from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import { useRouter } from "next/router"
+
+export enum ApplicationsFilterEnum {
+  All = 0,
+  Lottery,
+  Open,
+  Closed,
+}
+interface ApplicationsCount {
+  total: number
+  lottery: number
+  open: number
+  closed: number
+}
+
+interface ApplicationsViewProps {
+  filterType: ApplicationsFilterEnum
+}
+
+const ApplicationsView = (props: ApplicationsViewProps) => {
+  const { applicationsService, listingsService, profile } = useContext(AuthContext)
+  const [applications, setApplications] = useState<AppWithListing[]>()
+  const [applicationsCount, setApplicationsCount] = useState<ApplicationsCount>()
+  const [loading, setLoading] = useState(true)
+  const [listLoading, setListLoading] = useState(true)
+  const [error, setError] = useState()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (profile && loading) {
+      pushGtmEvent<PageView>({
+        event: "pageView",
+        pageTitle: `My Applications - ${ApplicationsFilterEnum[props.filterType]}`,
+        status: UserStatus.LoggedIn,
+      })
+      applicationsService
+        .list({ userId: profile.id })
+        .then((apps) => {
+          apps?.items?.length > 0 ? setApplications(apps.items) : setLoading(false)
+        })
+        .catch((err) => {
+          console.error(`Error fetching applications: ${err}`)
+          setError(err)
+          setLoading(false)
+        })
+    }
+  }, [profile, applicationsService])
+
+  useEffect(() => {
+    if (!applications || (applications && !listLoading)) return
+
+    void Promise.all(
+      applications?.map(async (app) => {
+        const retrievedListing = await listingsService.retrieve({
+          id: app?.listings.id,
+        })
+        app.fullListing = retrievedListing
+        return app
+      })
+    )
+      .then((res) => {
+        setListLoading(false)
+        const displayApplications = []
+        const total = res.length
+        let open = 0,
+          closed = 0,
+          lottery = 0
+        res.forEach((app) => {
+          if (app.fullListing?.status === ListingsStatusEnum.active) {
+            open++
+            if (props.filterType === ApplicationsFilterEnum.Open) displayApplications.push(app)
+          } else if (app.fullListing?.lotteryStatus === LotteryStatusEnum.publishedToPublic) {
+            lottery++
+            if (props.filterType === ApplicationsFilterEnum.Lottery) displayApplications.push(app)
+          } else {
+            closed++
+            if (props.filterType === ApplicationsFilterEnum.Closed) displayApplications.push(app)
+          }
+        })
```

Comment:
I don't know if this is the spot that is causing the issue. But the placement of applications seems to be incorrect. For example in this screenshot the "Accepting applications" shows a count of 1 but the application is showing up in the "Applications closed"

![image](https://github.com/user-attachments/assets/49aec7f0-8c06-4122-bafa-2fadc3725dac)

Reply:
This has been fixed